dadguide/database_context.py

```diff
@@ -44,10 +43,6 @@ def get_leader_skill_query(self, leader_skill_id: int):
         return self.database.select_one_entry_by_pk(
             leader_skill_id, DgLeaderSkill, db_context=self)
 
-    def get_awoken_skill(self, awoken_skill_id):
```

Comment:
deleted because..... no remaining users?

Reply:
yeah I did a check in pycharm